pages/Documents/Solutions/MobileAppMessaging/add-messaging-to-your-app.md

```diff
@@ -24,15 +24,21 @@ When you add messaging to your app, you enable your consumers to communicate wit
 
 * Optimized for network bandwidth and battery usage.
 
-<img src="img/inapp1.png" alt="InApp" style="max-width:230px;max-height:700px;"> <img src="img/inapp2.png" style="max-width:230px;max-height:700px;"> <img src="img/inapp3.png" alt="InApp" style="max-width:230px;max-height:700px;">
+<img src="img/inapp1.png" alt="InApp" style="max-width:230px;max-height:700px;"> 
+<img src="img/inapp2.png" alt="Add Messaging to Your App" style="max-width:230px;max-height:700px;">
+<img src="img/inapp3.png" alt="InApp" style="max-width:230px;max-height:700px;">
 
 ### Prerequisites
 
 To begin, you need:
 
 * Conversational Cloud account with messaging enabled.
 
-  <div class="attn-alert">Contact your LivePerson representative to enable messaging on your account.  <b>Tip:</b> If you see 'All Conversations’ when you log in, your account has messaging enabled.</div>
```

Comment:
(Good catch here! This does look like an alert and a tip. Interesting combination :)